internal/services/sources_service_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/RHEnVision/provisioning-backend/internal/clients/sources/stubs"
 	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
 	"github.com/stretchr/testify/assert"
+	"k8s.io/utils/pointer"
```

Comment:
This is literally one-liner and we would pull the largest Go project in the world just because of this line:

https://github.com/kubernetes/utils/blob/3a6ce19ff2f9/pointer/pointer.go#L159

Can you simply copy and paste the function into our own package instead? Please avoid utils package, name it simply "pointer".

Reply:
Thank you for noticing! :)